blocks.js

```diff
@@ -122,10 +122,13 @@ function showControls( node ) {
 
 	// show/hide block-specific block controls
 	var kinds = getTypeKinds( blockType );
-	var kindClasses = kinds.map( function( kind ) {
+	kinds.map( function( kind ) {
 		return 'is-' + kind;
-	} ).join( ' ' );
-	blockControls.className = 'block-controls ' + kindClasses;
+	} )
+	.forEach( function( className ) {
```

Comment:
For consistency, could you join the two above lines into a single `} ).forEach( …`?